spec/models/pillar_spec.rb

```diff
@@ -6,71 +6,14 @@
   it { is_expected.to validate_presence_of(:pillar) }
   it { is_expected.to validate_presence_of(:value) }
 
-  describe "Validator" do
-    context "on protected values" do
-      it "does not allow non-ascii digits" do
-        name = described_class.all_pillars[:city]
-        pillar = described_class.create pillar: name, value: "Nürnberg"
-
-        expect(pillar.errors.messages).to eq(value: ["contains invalid characters"])
-      end
-
-      it "does allow alphabetical" do
-        name = described_class.all_pillars[:city]
-        pillar = described_class.create pillar: name, value: "Nuernberg"
-
-        expect(pillar.errors.messages).to be_empty
-      end
-    end
-
-    context "on non-protected values" do
-      it "does allow non-ascii digits" do
-        pillar = described_class.create pillar: "something.foo", value: "Nürnberg"
-
-        expect(pillar.errors.messages).to be_empty
-      end
-    end
-  end
-
   describe "#apply" do
     let(:settings_params) do
       {
-        apiserver:    "something",
-        city:         "Nuremberg",
-        company_name: "SUSE Linux GmbH",
-        company_unit: "Research and development",
-        country:      "DE",
-        dashboard:    "something",
-        email:        "containers@suse.de",
-        state:        "Bavaria"
+        dashboard: "something",
```

Comment:
even though it's not technically important, and it has been like that before, i'd rather keep some real world values here that make sense (IP address or fqdn)